src/atoms/Link/Link.web.js

```diff
@@ -1,3 +1,74 @@
-const Link = () => null;
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import { getColor } from '../../_helpers/theme';
+
+const styles = {
+  color({ disabled, theme }) {
+    if (disabled) {
+      return getColor(theme, 'primary.800');
+    }
+    return getColor(theme, 'primary.600');
+  },
+  pointerEvents({ disabled }) {
+    if (disabled) {
+      return 'none';
+    }
+    return '';
+  },
+  hover: {
+    color({ disabled, theme }) {
+      if (disabled) {
+        return getColor(theme, 'primary.800');
+      }
+      return getColor(theme, 'primary.600');
+    },
+  },
+  focus: {
+    color({ disabled, theme }) {
+      if (disabled) {
+        return getColor(theme, 'primary.800');
+      }
+      return getColor(theme, 'primary.600');
+    },
+  },
+};
+
+const Anchor = styled(Text)`
+  color: ${styles.color};
+  text-decoration: none;
+  cursor: pointer;
+  pointer-events: ${styles.pointerEvents};
+  &:hover {
+    color: ${styles.hover.color};
+  }
+  &:focus {
+    outline: none;
+    text-decoration: underline;
+    color: ${styles.focus.color};
+  }
+`;
+
+const Link = ({ size, children, color, onClick, ...props }) => {
+  return (
+    <Anchor as="a" size={size} {...props} onClick={onClick}>
```

Comment:
We shouldn't pass down all the props like ```{...props}``` this way we'll lose control over the component and the consumer will start passing whatever they want